server/src/routes/api/v1/reviewsRouter.js

```diff
@@ -0,0 +1,9 @@
+import express from "express"
+import { Review } from "../../../models/index.js"
+import showsReviewRouter from "./showsReviewsRouter.js"
+
+const reviewsRouter = new express.Router()
+
+reviewsRouter.use("/shows/:id", showsReviewRouter)
```

Comment:
a showReviewsRouter should be mounted in the `shows` router, not in the `reviewsRouter`. let me know when youve started tackling some of this and we can go over it if necessary